### PR TITLE
TinyMCE5

### PR DESCRIPTION
**Pull request by JobDoesburg**
### Summary
Updates our tinymce dependency

### How to test
1. View event admin change
2. View an event details page

### CODE REVIEW

File: website/thaliawebsite/settings.py [hunk 1]
```diff
@@ -682,11 +682,10 @@ def from_env(
 PHOTO_UPLOAD_SIZE = 2560, 1440
 
 # TinyMCE config
-TINYMCE_JS_URL = "/static/tinymce/js/tinymce/tinymce.min.js"
+TINYMCE_JS_URL = "/static/tinymce/tinymce.min.js"
 
 TINYMCE_DEFAULT_CONFIG = {
     "selector": "textarea",
-    "theme": "modern",
     "plugins": "link image paste code contextmenu",
```
Comment (se-bastiaan) [inline website/thaliawebsite/settings.py:689]: I don't believe the image plugin is still working

Comment (se-bastiaan) [reply website/thaliawebsite/settings.py:689]: Screenshot 2022-02-02 at 20 37 42
Missing the button mostly

Comment (pingiun) [reply website/thaliawebsite/settings.py:689]: Before:
image

After:
image

File: website/thaliawebsite/settings.py [hunk 2]
```diff
@@ -682,11 +682,10 @@ def from_env(
 PHOTO_UPLOAD_SIZE = 2560, 1440
 
 # TinyMCE config
-TINYMCE_JS_URL = "/static/tinymce/js/tinymce/tinymce.min.js"
+TINYMCE_JS_URL = "/static/tinymce/tinymce.min.js"
```
Comment (se-bastiaan) [inline website/thaliawebsite/settings.py:685]: I don't believe this file is used

Comment (se-bastiaan) [reply website/thaliawebsite/settings.py:685]: wait no it is, but we still have a file in our codebase with an old version

Comment (se-bastiaan) [reply website/thaliawebsite/settings.py:685]: and this is the default value basically

**Review by pingiun — APPROVED** [review]

**Pull request merged**